moped-editor/src/views/projects/projectView/ProjectPhases.js

```diff
@@ -194,6 +194,11 @@ const ProjectPhases = ({ projectId, data, refetch }) => {
     refetch().then(() => setEditPhase(null));
   };
 
+  // Open activity edit modal when double clicking in a cell
+  const doubleClickListener = (params) => {
```

Comment:
Part of me thought this could be extracted as a helper since it is the same function in two separate components. I would feel stronger if this was in every table, though.